File: Exercices/Java/exo-java/src/exo4_5.java
import java.util.Scanner;
public class exo4_5 {

	public static void main(String[] args) {
		String input;
		int age;
		
		Scanner saisie = new Scanner(System.in);
		
		System.out.println("Veuillez entrer le sexe (saisir homme ou femme) : ");
		input = saisie.next();
		
		if (!"homme".equals(input) && !"femme".equals(input))
		{
			System.out.println("Erreur de saisie, veuillez recommencer.\n");
			System.exit(0);
		}

		System.out.println("Veuillez entrer l'age : ");
		age = saisie.nextInt();
		
		if (input.equals("homme") && age >= 20)
		{
			System.out.println("Imposable");
		}
		
			else if (input.equals("femme") && (age > 18 && age < 35) )
			{
				System.out.println("Imposable");
			}
		
			else
			{
				System.out.println("Non Imposable");
			}
		
		saisie.close();
	}

}

File: Exercices/Langage C/exo4_5.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int age;
    char input;

    printf("Saisir le sexe (h ou f) : ");
    scanf("%c",&input);

    if (input != 'h' && input != 'f')
    {
        printf("Erreur de saisie, veuillez recommencer.\n");
        return 0;
    }

    printf("Saisir l'age : ");
    scanf("%d",&age);

    if (input == 'h' && age >= 20)
		{
			printf("Imposable");
		}

			else if (input == 'f' && (age > 18 && age < 35) )
			{
				printf("Imposable");
			}

			else
			{
				printf("Non Imposable");
			}

    return 0;
}
